paperless/downloadfile.go

```diff
@@ -0,0 +1,80 @@
+package paperless
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var documentselection int
+
+func (p Paperless) setFullDownloadURL(location string) string {
+	switch p.UseHTTPS {
+	case true:
+		return "https://" + p.Hostname + location
+	case false:
+		return "http://" + p.Hostname + location
+	}
+	return ""
+}
+
+func (p Paperless) writeFile(document Document) {
+	downloadURL := p.setFullDownloadURL(document.DownloadURL)
```

Comment:
We can remove this whole function and instead use similar functionality in `paperless/paperless.go`: 

```suggestion
func (p Paperless) writeFile(document Document) {
	downloadURL := p.DownloadString(document.DownloadURL)
```

and add into `paperless/paperless.go`:
```golang
// DownloadString returns a string URL to fetch a document from the API.
func (p Paperless) DownloadString(documentPath string) string {
	if !p.UseHTTPS {
		return fmt.Sprintf("http://%v:%v%v", p.Hostname, p.Port, documentPath)
	}
	return fmt.Sprintf("https://%v:%v%v", p.Hostname, p.Port, documentPath)
}
```

Reply:
👍